src/apps/statements/service/storeStatements/queriables/getMetadataFromStatement/index.ts

```diff
@@ -0,0 +1,13 @@
+import Statement from '../../../../models/Statement';
+import { getDurationMetadata } from './getDurationMetadata';
+import { getSequencingMetadata } from './getSequencingMetadata';
+
+export default (statement: Statement): {readonly [key: string]: any} => {
+  const durationMetadata = getDurationMetadata(statement);
+  const sequencingMetadata = getSequencingMetadata(statement);
+
+  return {
+    ...(durationMetadata ? durationMetadata : {}),
+    ...(sequencingMetadata ? sequencingMetadata : {}),
```

Comment:
Nice. Just out of interest, why did you decide to return `false` from the `getDurationMetadata` and `getSequencingMetadata` functions rather than returning an empty object? Seems like that could have saved us having this logic here, I do appreciate that it makes the tests slightly slower because we have to use `assert.deepEquals` rather than the simpler and quicker `assert.equals`.

Reply:
Well, at that moment I didn't know that we can return multiple metadata combined into one object. So yeah now it's better to return empty object and use spread operator

Reply:
Okay nice, thanks Vadim.